### PR TITLE
Add benchmark suite for exposure module

### PR DESCRIPTION
**Pull request by abhishekarya286**
## Description
I have added benchmarks for `exposure.rescale_intensity()` and `exposure.equalize_adapthist()`.

## Checklist
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [x] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests

[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]

## References
[If this is a bug-fix or enhancement, it closes issue # ]
[If this is a new feature, it implements the following paper: ]

## For reviewers

(Don't remove the checklist below.)

- [x] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by abhishekarya286** [comment]
Hi @jni @soupault. Please review. The time these functions take is 200ms(rescale) and 450ms(adapthist) approximately.

**Comment by abhishekarya286** [comment]
Hi @jni. I have updated this PR to run benchmarks just once. Please review.

File: benchmarks/benchmark_exposure.py [hunk 1]
```diff
@@ -12,8 +12,15 @@ class ExposureSuite:
     def setup(self):
         self.image = img_as_float(data.moon())
         self.image = rescale(self.image, 2.0, anti_aliasing=False)
+        # for Contrast stretching
+        self.p2, self.p98 = np.percentile(self.image, (2, 98))
 
     def time_equalize_hist(self):
-        # Running it 10 times to achieve significant performance time.
-        for i in range(10):
-            result = exposure.equalize_hist(self.image)
```
Comment (emmanuelle) [inline benchmarks/benchmark_exposure.py]: why did you get rid of the for loop?

Comment (jni) [reply benchmarks/benchmark_exposure.py]: @emmanuelle turns out asv does this kind of averaging by default.

**Comment by hmaarrfk** [comment]
@scikit-image/core I just merge master back into this one. Does anybody care for getting this cool contribution in?

**Comment by hmaarrfk** [comment]
Thanks @abhishekarya286, I took the liberty to merge master back into this to hopefully get somebody else's approval so we can move forward with this great work!

**Comment by hmaarrfk** [comment]
Lets just merge then?

**Comment by jni** [comment]
Yep go for it!

**Pull request merged**